state-chain/runtime/src/migrations/remove_aborted_broadcasts.rs

```diff
@@ -0,0 +1,85 @@
+use cf_chains::instances::{
+	ArbitrumInstance, BitcoinInstance, EthereumInstance, PolkadotInstance, SolanaInstance,
+};
+use cf_runtime_upgrade_utilities::VersionedMigration;
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+use pallet_cf_broadcast::AbortedBroadcasts;
+use sp_runtime::DispatchError;
+
+use crate::*;
+
+pub type AllInstancesMigration = (
+	// Only Eth and Arb have stale aborted broadcasts
+	VersionedMigration<
+		pallet_cf_broadcast::Pallet<Runtime, EthereumInstance>,
+		EthereumMigration,
+		9,
+		10,
+	>,
+	VersionedMigration<
+		pallet_cf_broadcast::Pallet<Runtime, ArbitrumInstance>,
+		ArbitrumMigration,
+		9,
+		10,
+	>,
+	VersionedMigration<pallet_cf_broadcast::Pallet<Runtime, SolanaInstance>, NoopUpgrade, 9, 10>,
+	VersionedMigration<pallet_cf_broadcast::Pallet<Runtime, PolkadotInstance>, NoopUpgrade, 9, 10>,
+	VersionedMigration<pallet_cf_broadcast::Pallet<Runtime, BitcoinInstance>, NoopUpgrade, 9, 10>,
+);
+
+// Stale aborted broadcasts on mainnet as of 23/09/2024
+const ETHEREUM_ABORTED_BROADCASTS: [BroadcastId; 5] = [3026, 3684, 3686, 11350, 11592];
+const ARBITRUM_ABORTED_BROADCASTS: [BroadcastId; 5] = [238, 239, 345, 423, 426];
```

Comment:
These values are only valid on mainnet. It's unlikely but possible that we unintentionally delete something on the other networks.

I think this would make more sense in the housekeeping migration:

- the operation is inherently idempotent: we can run it multiple times and the end-effect will always be the same. 
- we didn't make any changes to the storage format
- we can also delete sisyphos and perseverance values 

Then we can avoid bumping the pallet storage version. 

I would also suggest just writing it in terms of the maximum id that we want to delete: ie. `aborted.retain(|id| id > highest_value_to_delete)`.

Reply:
ah, didnt know "housekeeping" was a thing. yeah makes sense.